src/url-loader/multi-url-resolver.ts

```diff
@@ -47,11 +47,20 @@ export class MultiUrlResolver extends UrlResolver {
    * Delegates to relative method on the first resolver which can resolve the
    * destination URL.
    */
+  relative(to: ResolvedUrl): PackageRelativeUrl;
+  relative(from: ResolvedUrl, to: ResolvedUrl, kind?: string): FileRelativeUrl;
   relative(fromOrTo: ResolvedUrl, maybeTo?: ResolvedUrl, kind?: string):
-      FileRelativeUrl {
+      FileRelativeUrl|PackageRelativeUrl {
+    const from = (maybeTo === undefined) ? undefined : fromOrTo;
+    const to = (maybeTo === undefined) ? fromOrTo : maybeTo;
     for (const resolver of this._resolvers) {
-      if (resolver.resolve((maybeTo || fromOrTo) as any)) {
-        return resolver.relative(fromOrTo, maybeTo, kind);
+      if (!resolver.resolve(this.brandAsPackageRelative(to))) {
```

Comment:
Should for undefined explicitly, rather than coercing string to boolean.

I mean, it should probably never resolve to an empty string, but it's easier for the reader and the VM to be explicit here I think.

Reply:
Ok.  Will check for undefined before merging.